Service/LinkedInLocationService.php

```diff
@@ -186,4 +193,19 @@ public function handleCompanyPageCreation(Location $location)
         $flashBagMsg .= '<li>Page: <a href="'.$page->getLink().'">'.$page->getDisplayName().'</a></li>';
         $this->channelWizard->set('flashBagMsg', $flashBagMsg);
     }
+
+    /**
+     * Removes the not assigned tokens.
+     */
+    public function cleanUpUnassignedTokens()
```

Comment:
@ghostika I'd suggest to put this method into the Token service of the OAuth client module, because it's basically garbage collection for the latter module.